olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -2,7 +2,7 @@ schema: olm.channel
 name: stable
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.27.0"
+  - name: "serverless-operator.v1.27.1"
   - name: "serverless-operator.v1.28.0"
-    replaces: serverless-operator.v1.27.0
-    skipRange: ">=1.27.0 <1.28.0"
+    replaces: serverless-operator.v1.27.1
+    skipRange: ">=1.27.1 <1.28.0"
```

Comment:
I thought this would still be `">=1.27.0 <1.28.0"`. I'm not sure if updating this will work.